src/Microsoft.Data.Sqlite.Core/Microsoft.Data.Sqlite.Core.csproj

```diff
@@ -0,0 +1,43 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\build\common.props" />
+  <PropertyGroup>
+    <AssemblyName>Microsoft.Data.Sqlite</AssemblyName>
+    <RootNamespace>$(AssemblyName)</RootNamespace>
+    <PackageId>$(MSBuildProjectName)</PackageId>
+    <Description>SQLite implementation of the System.Data.Common provider model.
```

Comment:
I know you removed the Summary tag a cleanup commit, but consider re-adding it. NuGet will issue build warnings for long descriptions like this. In general, I'm in favor of eliminating warnings.